### ISSUE TITLE
[Feature Request] More Customizeable Totem Bar

### ISSUE THREAD

**Issue by Wastingmoney** [tukui-org/ElvUI#323]
**Is your feature request related to a problem? Please describe.**
So now where we have the good Totem Placement in WrathC. Im first time using the TotemBar now but since I've Binded my 3 different Sets and the Recall seperatly the buttons on the Totem Bar are feeling forced to me.
Not really an problem in the UI. Rather a personal issue (maybe some others find this useful tho)

**Describe the solution you'd like**

Just make a small option to hide the Both the Summoning Totem and the Recall Totem Buttons leaving the Totem Bar with just the pure Totems.

**Describe alternatives you've considered**
Theres an Addon kinda doing what I want. But having to have a whole Addon feels kind of overkill for how simple to solution would be in ElvUI itself

**Additional context**
< Add any other context or screenshots about the feature request here. >
TL;DR love to be able to go from this
![image](https://user-images.githubusercontent.com/70923476/188339404-6bc502a7-0988-4007-b5d2-1f7029279eea.png)
to this (achieved with /run xx:hide() )
![image](https://user-images.githubusercontent.com/70923476/188339410-d78add32-7102-4d32-8dde-2ff7affd0ea3.png)

Thanks in Advance :)
